client/src/app/s-degree-plan/s-degree-plan.component.html

```diff
@@ -24,7 +24,7 @@
           <!-- Semester Cards -->
           <div *ngFor="let semester of year.semesters" class="semester-card">
             <div class="semester-card-term-and-status">
-              <span>{{ semester.term }} {{ semester.year }}</span>
+              <span>{{ semester.term | uppercase }} {{ semester.year }}</span>
```

Comment:
Need to add an ngIf to the term and year in the span. As well as on the line 18 for the year start and beginning

Reply:
Added